### DISCUSSION TITLE
I am not able to inject styles using injectStylesUrls, how ever inline styles using injectStyles works

### DISCUSSION THREAD

**Discussion by sunilrathore24** [nolimits4web/swiper#6374] [Q&A]
I am not able to inject styles using injectStylesUrls, how ever inline styles using injectStyles works

I am using a swiper element with angular 15, 
trying to override swiper css, the below code works fine

this.swiperParams = {
    injectStyles: [
          `
           :host(.my-swiper) {
              background-color: red;
              color: blue;
            }
            `
        ]}

but if I copy the same code in a file and use that file using
injectStylesUrls, it doesn't work.

stackblitz example...
https://stackblitz.com/edit/angular-6ftpvm?file=src%2Fmain.ts

**Comment by awacode21** [comment]
i am dealing with the same issue. i am using swiper element with nuxt.
I try to use Core Version & Modules. But when trying to inject the style urls for the single modules i always get a 500.

did you manage to solve this in the past? Or does it make sense to create a bug report?

**Reply by sunilrathore24** [reply]
I have fixed it by using the parts option

https://swiperjs.com/element#parts
